internal/rbd/clone.go

```diff
@@ -46,7 +46,10 @@ import (
 func (rv *rbdVolume) checkCloneImage(ctx context.Context, parentVol *rbdVolume) (bool, error) {
 	// generate temp cloned volume
 	tempClone := rv.generateTempClone()
+	defer tempClone.Destroy()
+
```

Comment:
nit: may be this extra line is not required

Reply:
not required, but with the extra line it makes it easier to read